go/common/crypto/slip10/slip0010.go

```diff
@@ -0,0 +1,93 @@
+// Package slip10 implements the SLIP-0010 private key derivation
```

Comment:
Nit: Rename `slip0010{,_test}.go` to `slip10{,_test}.go`

And also in the commit message (i.e. "common/crypto/slip10: ...").